src/pages/WalletsItemAddresses/WalletsItemAddressesView.js

```diff
@@ -27,21 +28,45 @@ export type Props = {|
 
 type StateProps = {|
   +addresses: Address[],
+  +ethBalance: ?BigNumber,
 |}
 
 export class WalletsItemAddressesView extends PureComponent<Props, StateProps> {
   constructor(props: Props) {
     super(props)
 
     this.state = {
+      ethBalance: null,
       addresses: this.deriveAddresses(props),
     }
   }
 
-  componentWillReceiveProps(nextProps: Props) {
+  async componentDidMount() {
+    this.requestETHBalance()
+  }
+
+  async componentWillReceiveProps(nextProps: Props) {
```

Comment:
This is deprecated method. We should not use it, or use `UNSAFE_` prefix if there is no way to make it.

https://reactjs.org/docs/react-component.html#unsafe_componentwillreceiveprops